app/routes.py

```diff
@@ -1,2 +1,196 @@
-from flask import Blueprint
 
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime, timedelta
+import os
+
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+# CUSTOMER
+
+@customers_bp.route("", methods = ["GET"])
+def customer_index():
+    customers = Customer.query.all()
+    customers_response = [customer.to_json() for customer in customers]
+    return make_response(jsonify(customers_response), 200)
+
+@customers_bp.route("", methods = ["POST"])
+def customers():
+    request_body = request.get_json()
+    if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    else:
+        new_customer = Customer(name=request_body["name"], 
+                                postal_code=request_body["postal_code"],
+                                phone=request_body["phone"])
+        new_customer.registered_at = datetime.now()
+        db.session.add(new_customer)
+        db.session.commit()
+        return make_response({"id": new_customer.id}, 201)
+
+@customers_bp.route("/<id>", methods = ["GET"])
+def get_one_customer(id):
+    customer = Customer.query.get_or_404(id)
+    return  make_response(customer.to_json(), 200)
```

Comment:
Great use of `get_or_404()`, to make this request more robust I suggest adding a description in case an invalid id is passed:

```suggestion
def get_one_customer(id):
    customer = Customer.query.get_or_404(id, description="id not found")
    return  make_response(customer.to_json(), 200)
```